packages/miniflare/src/workers/core/proxy.worker.ts

```diff
@@ -69,7 +69,15 @@ function isPlainObject(value: unknown) {
 	);
 }
 function objectContainsFunctions(obj: Record<string, unknown>): boolean {
-	for (const [, entry] of Object.entries(obj)) {
+	const propertyNames = Object.getOwnPropertyNames(obj);
+	const propertySymbols = Object.getOwnPropertySymbols(obj);
+	const properties = [...propertyNames, ...propertySymbols];
```

Comment:
@petebacondarwin this is the fix regarding symbol properties mentioned in the changeset

Reply:
Yes exactly, thanks @RamIdeas 🙂 :+1: